linter.py

```diff
@@ -41,15 +44,16 @@ class Codeclimate(Linter):
     error_stream = util.STREAM_BOTH
     word_re = None
 
+    root = None
+    path = None
+
     def cmd(self):
-        """Construct a cmd to provide a relative path to 'codeclimate analyze'."""
-        result = ['codeclimate', 'analyze', '-e', 'structure', '-e', 'duplication']
```

Comment:
The reason why the flags are defined here is because CodeClimate will run extra engines if they're defined in `.codeclimate.yml` - which I don't need them to in my local environment because I have other linters for them (e.g. Rubocop)

Reply:
Hi @schuylr 

Thank you very much for your response.

As far as I can see in my tests, and I experienced the behavior of SublimeLinter, arguments take place in the SublimeLinter settings. They will be applied either of global settings or project-specific settings. I've tested both. If you set for instance 

```json
"linters": {
  "codeclimate": {
    "args": [
        "-e",
        "structure",
        "-e",
        "duplication"
      ]
  }
}
```

in the global settings, codeclimate will ignore the `.codeclimate.yml` in the project root because it will be executed with arguments (`$ codeclimate analyze path/to/file.ext -e structure -e duplication`)

I suggest, to take this way of using the SublimeLinter settings as it is designed, instead of "hard coding" the arguments.